src/jit/lsra.cpp

```diff
@@ -8450,7 +8424,9 @@ void LinearScan::resolveRegisters()
         printf("Trees after linear scan register allocator (LSRA)\n");
         compiler->fgDispBasicBlocks(true);
     }
-    verifyFinalAllocation();
+
+// TODO(pdg): re-enable verification once it's moved to LIR, as it will almost certainly be useful.
+// verifyFinalAllocation();
```

Comment:
Yes, I think that would be good, as it is functionality I would hate to be without.

---
In reply to: [75239161](https://github.com/dotnet/coreclr/pull/6689#discussion_r75239161) [](ancestors = 75239161)